src/jsx/components/busted_summary.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+import {InputInfo} from './input_info';
+
+var BUSTEDSummary = React.createClass({
+  render: function(){
+    return (
+      <div className="row" id="summary-div">
+        <div className="col-md-8">
+          <h3 className="list-group-item-heading">
+            <span id="summary-method-name">BUSTED summary</span>
+          </h3>
+          <div className="main-result">
+            <p>
+              BUSTED <strong className="hyphy-highlight">found {this.props.test_result.statement}</strong><sup>†</sup> of episodic diversifying selection (LRT P-value={this.props.test_result.p}) in the selected foreground of your phylogeny. Therefore, there is evidence that at least one site on at least one foreground branch has experienced diversifying selection. 
+            </p>
+            <hr/>
+            <p>
+              <small>
+                <sup>†</sup> LRT, p ≤ 0.05.
+                <br/>
+              See <a href="http://hyphy.org/methods/selection-methods/#busted">here</a> for more information about the BUSTED method.
```

Comment:
We may want to use `//` for links so that the browser selects which protocol to use.